airbyte-integrations/connectors/source-github/source_github/schemas/workflow_jobs.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["integer"]
+    },
+    "run_id": {
+      "type": ["integer"]
+    },
+    "run_url": {
+      "type": ["string"]
+    },
+    "run_attempt": {
+      "type": ["null", "integer"]
+    },
+    "node_id": {
+      "type": ["string"]
+    },
+    "head_sha": {
+      "type": ["string"]
+    },
+    "url": {
+      "type": ["string"]
+    },
+    "html_url": {
+      "type": ["string"]
+    },
+    "status": {
+      "type": ["string"]
+    },
+    "conclusion": {
+      "type": ["string"]
+    },
+    "started_at": {
+      "type": ["string"]
+    },
+    "completed_at": {
+      "type": ["string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "steps": {
+      "type": "array",
```

Comment:
I guess you have tested that the array is acceptable as input, it will probably be a repeated field on the destination

Reply:
What do you mean. I defined the steps following github's schema definition